ext/RMagick/rmagick.h

```diff
@@ -121,6 +128,13 @@ typedef PixelPacket PixelColor;
 //! Montage
 typedef struct
 {
+    struct _change
```

Comment:
Why do we need the `_change` here? Is this a convention for clarity?

Reply:
it need to mark the properties that apply the changed. it is just namespace.

If you like flat style, it will be as following.
```
int change_background_color;
int change_border_color;
int change_gravity;
```

Reply:
I see. Thanks for the explanation. I think what you have is good.